samples/flickr/build/reports/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@khorozco It looks like you added this file accidentally, you can remove it with:

git rm samples/flickr/build/reports/checkstyle/checkstyle.xml

And then amend this commit with:

git commit --amend

Finally when you try to push, you will see an error because you've modified history, but you can work around that using --force:

git push origin master --force

Force pushing to your personal remote is ok because you can be pretty confident no one other than you is going to be working off of it. You see a warning because on shared repositories it's hard to contribute if history isn't linear.